PWM_NativePin.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
Just do that for .h files. Because:

`#include "some.h"` is EXACTLY like cut-n-paste of the file.

If `#include "some.h"` is also used in some other place, you've cut-n-pasted twice. Which is usually wrong.

`#pragma once` prevents more cut-n-paste.